src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -624,6 +641,7 @@ void quorum_cop::block_add(
         const cryptonote::block& block, const std::vector<cryptonote::transaction>& txs) {
     process_quorums(block);
     uint64_t const height = block.get_height() + 1;  // chain height = new top block height + 1
+    m_l2_height = block.l2_height;
```

Comment:
We don't need to store this, and likely don't want to use the value in the block anyway (because we could be behind, or the chain could stall, or a malicious quorum could lie in the block header), but rather we can just grab the latest value out of the L2Tracker when needed via `m_core.l2_tracker().get_latest_height()`.